### PR TITLE
Cherry-pick PR: #2478

### PR DESCRIPTION
**Pull request by camilamacedo86**
Cherry-pick PR: #2478

### CODE REVIEW

File: doc/proposals/kubernetes-1.17.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ creation-date: 2019-12-19
 last-updated: 2019-12-19
 status: implementable
 see-also:
-  - https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.17.md#changes
```
Comment (joelanford) [inline doc/proposals/kubernetes-1.17.md]: We should put the link fixes in a separate PR.

**Comment by camilamacedo86** [comment]
ignorre
